src/helpers/__tests__/ResolveRundownIntoPlaylist.spec.ts

```diff
@@ -24,14 +31,14 @@ function createUnrankedSegment(num: number, backTime?: string): UnrankedSegment
 				cumeTime: '',
 				backTime,
 			}),
-			meta: {},
-			cues: [],
-			body: '',
+			meta: meta ?? {},
+			cues: cues ?? [],
+			body: body ?? '',
 		}),
 	})
 }
 
-function createContinuitySegment(num: number, backTime?: string): UnrankedSegment {
+function createContinuitySegment(num: number, { backTime, cues, meta, body }: SegmentOptions = {}): UnrankedSegment {
```

Comment:
Why not write "segmentOptions: SegmentOptions" instead of "{backTime, cues, meta, body}: SegmentOptions"?

Reply:
I chose to unwrap the attributes, such that the unwrapped attributes explicitly tells which parts of SegmentOptions the function depends on.